lib/views/backend_v1.2/spree/admin/payments/source_forms/_paypal_braintree.html.erb

```diff
@@ -0,0 +1,16 @@
+<% id = payment_method.id %>
+
+<fieldset class="no-border-bottom braintree-hosted-fields" data-braintree-hosted-fields data-id="braintree" data-payment-method-id="<%= id %>">
```

Comment:
Is `data-id="<%= id %>"` vs `data-id="braintree"` the only difference between these files? Can we move that out into ruby land where it's easier to understand and share the same partial?